AsyncDisplayKit/ASPagerNode.m

```diff
@@ -14,7 +14,6 @@
 #import "ASDelegateProxy.h"
 #import "ASDisplayNode+Subclasses.h"
 #import "ASPagerFlowLayout.h"
-#import "ASCollectionView+Undeprecated.h"
```

Comment:
does the absence of this line mean that the collectionView methods cannot be called?